velox/functions/sparksql/specialforms/SparkCastHooks.cpp

```diff
@@ -42,6 +42,15 @@ Expected<int32_t> SparkCastHooks::castStringToDate(
       removeWhiteSpaces(dateString), util::ParseMode::kNonStandardCast);
 }
 
+Expected<float> SparkCastHooks::castStringToReal(const StringView& data) const {
+  return util::Converter<TypeKind::REAL>::tryCast(data);
```

Comment:
Does the implementation have any code specific to Presto? If so, that code is no longer needed and can be removed.

Reply:
No, the code in util::Converter<TypeKind::REAL>::tryCast(const StringView&) is pretty general. It just calls folly::tryTo() and handles errors.

Reply:
Got it. Thank you for confirming.